### DISCUSSION TITLE
Help with understanding output

### DISCUSSION THREAD

**Discussion by Abbie-Williams** [FrederickHuangLin/ANCOMBC#30] [Q&A] [answered]
Hi, 

I've been playing around with ANCOM-BC and I think it could be incredibly useful for analysis I'm doing.  I've been doing some reading, but I am still a little confused about the formula and variables in the output. In my metadata, I have columns for infection (Case / Control), timepoints and isolates. I'm interested to see if taxon are differentially abundant between case and controls overall, as well as isolates and timepoints. The formula I used was "Infection + Timepoints + Isolate", and I specified the group as "Infection". 

There are differentially abundant taxa in the output. However, I'm not sure whether these are more abundant in the cases or controls. in the differentially abundant table, the first column is "InfectionControl". This makes sense to me, as when looking at the log fold change values, those that are abundant in the control will less abundant in the cases and vice versa. However, the following columns are just "TimepointsA", "Timepoints B", "Timepoints C" etc. How do I know if features that are differentially abundant in the timepoints relate to cases or controls? I have a feeling I've fundamentally misunderstood something, but statistics has never been my strong point. 

The package looks really helpful and I can't wait to properly use it, 

Abbie

**Comment by FrederickHuangLin** [comment, marked as answer]
Hi Abbie,

Thanks for your question!

Your question is related to the interpretation of the coefficients.

Since the ANCOM-BC methodology is based on a linear model framework, let me illustrate it in a simple linear regression model. Suppose we have two variables, `group` and `timepoint`, each has two levels (`case` vs. `control`, and `A` vs. `B`, respectively). The underlying model can be written as:

Y = b0 + b1 * I(group = case) + b2 * I(timepoint = B) + e
where I(.) is an indicator function.

To interpret `b1`, it is the effect size for a subject belonging to `case` as compared to a subject belonging to `control` **given** they are at the same time point.

Similar, `b2` should be interpreted as the effect size for a subject at time point `B` as compared to a subject at time point `A` **given** they are at the same group.

For details, you can refer to this [post](https://web.ma.utexas.edu/users/mks/statmistakes/regressioncoeffs.html) or the section of multivariable linear regression in related text books.

Back to your question, if you are interested in whether a taxon is differentially abundant between `case` and `control` in any time point, you can still go to the `InfectionControl` column in your output, but you need to interpret it in an appropriate language (conditioning on the same time point, etc.).

Best,
Huang
